tests/test_runner.py

```diff
@@ -51,20 +51,20 @@ def test_get_runner_ansible_test_local(temp_config_file):
         assert runners.get_runner(cfg) == runners.LocalAnsibleTestRunner
 
 
-def test_get_runner_pulp(temp_config_file):
+def test_get_runner_local_image(temp_config_file):
     with open(temp_config_file, 'w') as f:
         f.write('[galaxy-importer]\nRUN_ANSIBLE_TEST = True\n'
-                'INFRA_PULP = True')
+                'ANSIBLE_TEST_LOCAL_IMAGE = True')
         f.flush()
         config_data = config.ConfigFile.load()
         cfg = config.Config(config_data=config_data)
         assert runners.get_runner(cfg) == runners.LocalImageTestRunner
 
 
-def test_get_runner_pulp_and_osd(temp_config_file):
+def test_get_runner_local_image_and_osd(temp_config_file):
```

Comment:
Having both `ANSIBLE_TEST_LOCAL_IMAGE` and `INFRA_OSD` as `True` conflict in terms of picking a Runner, so this test changes nature a bit to see which config option wins, probably should now be named `test_osd_config_wins_over_local_image()`